File: codeWars/sumOfDigits.js
function getSumOfDigits(integer) {
    let sum = 0;
    let digits =  Math.floor(integer).toString();

    for(let ix = 0; ix < digits.length; ix++) {
      sum = sum + Number(digits[ix]);
    }
    return sum;
}

console.log(getSumOfDigits(123));

File: codeWars/sum_of_digits.py
def get_sum_of_digits(num):
    sum = 0
    digits = str(num)
    array = []

    for i in digits:
        array.append(int(i))

    for x in array:
        sum = sum + x
    return sum

print(get_sum_of_digits(1234))